server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -93,28 +113,46 @@ export class DeliLambdaFactory
 		};
 
 		let gitManager: IGitManager;
-		let document: IDocument;
+		let document: IDocument = {} as IDocument;
 
 		try {
+			const documentCacheKey = invalidDocumentCacheKey(tenantId, documentId);
+			const sessionCacheKey = invalidSessionCacheKey(tenantId, documentId);
+			const invalidDocumentCacheEntryExists = this.invalidDocumentOrSessionCacheEnabled ? this.invalidDocumentOrSessionCache.has(documentCacheKey) : false;
```

Comment:
nit, to simply, do `this.invalidDocumentOrSessionCacheEnabled && this.invalidDocumentOrSessionCache.has(documentCacheKey)`?